backend/src/test/java/com/benine/backend/http/PresetCreationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package com.benine.backend.http;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.CameraController;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.database.DatabasePreset;
+
+import org.junit.Assert;
+
+public class PresetCreationHandlerTest {
+
+  @Test
+  public void testGetCameraPositions() throws CameraConnectionException {
+    CameraController controller = mock(CameraController.class);
+    PresetCreationHandler handler = new PresetCreationHandler(controller);
+    IPCamera ipcamera = mock(IPCamera.class);
+    when(ipcamera.getFocusPos()).thenReturn(33);
+    when(ipcamera.getIrisPos()).thenReturn(50);
+    when(ipcamera.getPosition()).thenReturn(new Position(0, 0));
+    when(ipcamera.getZoomPosition()).thenReturn(100);  
+    when(ipcamera.isAutoFocusOn()).thenReturn(true);
+
```

Comment:
As Jochem pointed out elsewhere, you can split this up into multiple (Prameterized) methods which will aid finding the problem if (one of the) the test(s) fail(s).